### DISCUSSION TITLE
use puma-dev error

### DISCUSSION THREAD

**Discussion by jisuanjixue** [templatus/templatus-inertia#498] [Q&A]
i am from https://github.com/ElMassimo/vite_ruby/discussions/215 , I follow the steps in the beginning, now i have new error:
Cannot access this site pingcrm-react.test has unexpectedly terminated the connection.
Please try the following:
check network connection
Check proxy servers and firewalls
ERR_CONNECTION_CLOSED

<img width="1096" alt="屏幕快照 2022-11-23 下午17 30 32 下午" src="https://user-images.githubusercontent.com/24429825/203512993-9e8ea841-b709-4a29-89c8-acd44032f8d4.png">

**Comment by ledermann** [comment]
I just see there is `pingcrm-react.test` mentioned in your bug report. Are you sure you have forked `templatus/templatus-inertia`? Templatus-Inertia uses Svelte, not React. And it's not PingCRM, which is an entirely different project (see https://github.com/ledermann/pingcrm). BTW: My PingCRM repo is not configured for `puma-dev`, it just runs on `localhost:3000`.

So, I'm confused. Is there a public repo that demonstrates the problem?

**Reply by jisuanjixue** [reply]
i forked (https://github.com/ledermann/pingcrm) and i see [templatus/templatus-inertia] use puma-dev, i just added it in pingcrm project

**Reply by ledermann** [reply]
PingCRM does not use `puma-dev` because it demonstrates Inertia and is NOT a template for new apps.

You can add `puma-dev`, but this requires some additional setup. To just learn Inertia I suggest not adding puma-dev.

**Reply by jisuanjixue** [reply]
@ledermann  this is my repo https://github.com/jisuanjixue/pingcrm_react, i think Is it a puma-dev certificate problem, or my local dns problem, i don't know

**Reply by jisuanjixue** [reply]
@ledermann  ok , I roll back my version
